src/dotnet/commands/dotnet-sln/LocalizableStrings.resx

```diff
@@ -165,4 +165,10 @@
   <data name="InRoot" xml:space="preserve">
     <value>Place project in root of the solution, rather than creating a solution folder.</value>
   </data>
-</root>
+  <data name="AddProjectSolutionFolderArgumentDescription" xml:space="preserve">
+    <value>The destination solution folder path to add the projects to.</value>
+  </data>
+  <data name="SolutionFolderAndInRootMutuallyExclusive" xml:space="preserve">
+    <value>The --solution-folder and --in-root options are mutually exclusive</value>
```

Comment:
I think we can probably use a slightly friendlier error message that doesn't use jargon like "mutually exclusive" which might confuse people unfamiliar with the term.

Perhaps this:

```suggestion
    <value>The --solution-folder and --in-root options cannot be used together; use only one of the options.</value>
```

What do you think?

Reply:
Sounds fine with me. I'm indifferent to texts, as long as the point gets across :)